### PR TITLE
fix: Make WorkstationCluster network/subnetwork required

### PR DESCRIPTION
**Pull request by jasonvigil**

### CODE REVIEW

File: apis/workstations/v1alpha1/workstationcluster_types.go [hunk 1]
```diff
@@ -56,12 +56,12 @@ type WorkstationClusterSpec struct {
 
 	// Immutable. Reference to the Compute Engine network in which instances associated
 	//  with this workstation cluster will be created.
-	NetworkRef *refs.ComputeNetworkRef `json:"networkRef,omitempty"`
+	NetworkRef refs.ComputeNetworkRef `json:"networkRef"`
```
Comment (justinsb) [inline apis/workstations/v1alpha1/workstationcluster_types.go:59]: Nit: I suggest using the `// +required` option to make this explicit

**Comment by justinsb** [comment]
LGTM.  Will approve and hold so you see the suggestion about `// +required`

/approve
/lgtm
/hold

File: apis/workstations/v1alpha1/workstationcluster_types.go (hunk 1 above)
Comment (jasonvigil) [reply apis/workstations/v1alpha1/workstationcluster_types.go:60]: fixed!

**Review by justinsb — APPROVED** [review]

**Pull request merged**